### PR TITLE
Removed step to select Screenshots checkbox

### LINKED ISSUE #3027: Simulate mobile CPU
**Link to the documentation page**:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/#simulate-a-mobile-cpu

**Description of the issue**:
Reading the docs on how to simulate mobile CPU, it's not clear why **Step 2** (_Select the Screenshots checkbox_) is required. I was able to test the [demo page](https://microsoftedge.github.io/Demos/devtools-performance-get-started/) to slow the down the CPU (4x/6x) successfully without needing Step 2.

I think the text below the screenshot which says (_The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes._) may also be not relevant anymore, as I was able to throttle the CPU to 6x slower

**Screenshots**:
image

**Microsoft Edge version**:
Version 120.0.2210.144 (Official build) (arm64)

### PR DESCRIPTION
**Pull request by pankajparashar**
Rendered article section for review: 
Internal, fully rendered:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/?branch=pr-en-us-3032#simulate-a-mobile-cpu
Public GitHub Markdown preview:
https://github.com/pankajparashar/edge-developer/blob/patch-4/microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md#simulate-a-mobile-cpu

Live/Before: 
https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/#simulate-a-mobile-cpu

Fixes #3027 

Tested the simulation with Screenshot unchecked and 6x slowdown:

https://github.com/MicrosoftDocs/edge-developer/assets/38640616/8cd0d609-f669-4332-a873-5d1cb21e9b24

AB#48649534

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 1]
```diff
@@ -59,15 +59,13 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
 1. In DevTools, open the **Performance** tool.
 
-1. Select the **Screenshots** checkbox.
-
 1. Click **Capture settings** (![Capture settings](./index-images/capture-settings-icon.png)).  DevTools reveals settings related to how it captures performance metrics.
 
 1. For **CPU**, select **4x slowdown**.  DevTools throttles your CPU so that it's 4 times slower than usual.
 
    ![CPU throttle](./index-images/capture-settings.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:66]: the screenshot shows the Screenshots checkbox selected, not cleared.  Clear it in the screenshot?  Is this checkbox  selected by default?

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 2]
```diff
@@ -59,15 +59,13 @@ Mobile devices have much less CPU power than desktops and laptops.  Whenever you
 
 1. In DevTools, open the **Performance** tool.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:61]:
```suggestion
1. Make sure the **Screenshots** checkbox is selected.

```
is the checkbox required to be selected?  if so, restore step, w/ this wording.

Comment (pankajparashar) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:61]: Yes, simulating a mobile CPU, has nothing to do with Screenshots. It doesn't matter whether it is enabled or not. Hence, Step 2 is not required, and hence, removed it from the doc.

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:61]: step isn't needed

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:66]: checkbox doesn't matter

**Review by mikehoffms — APPROVED** [review]
Approved per Writer/Editor review

**Review by captainbrosset — APPROVED** [review]

**Comment by captainbrosset** [comment]
Thank you @pankajparashar!

**Pull request merged**